runtime/oti/vmconstantpool.xml

```diff
@@ -190,7 +190,8 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<fieldref class="java/lang/StackTraceElement" name="fileName" signature="Ljava/lang/String;"/>
 	<fieldref class="java/lang/StackTraceElement" name="lineNumber" signature="I"/>
 	<fieldref class="java/lang/StackTraceElement" name="source" signature="Ljava/lang/Object;"/>
-	<fieldref class="java/lang/Thread" name="threadRef" signature="J" cast="struct J9VMThread *"/>
+	<fieldref class="java/lang/Thread" name="eetop" signature="J" cast="struct J9VMThread *" flags="opt_ojdkThreadSupport"/>
```

Comment:
Can you make eetop an alias of threadRef so all the code doesn't need to change?

Reply:
Alias works w/ newly added `VMCPTool` NOT operator `!`.